src/components/GoToTopButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { memo } from "react"
+import { IoChevronUpSharp } from "react-icons/io5"
+
+import { Button } from "./Buttons"
+
+const scrollToTop = () => {
+  window.scrollTo({ top: 0, behavior: "smooth" })
+}
+
+export const GoToTopButton = memo(() => {
+  return (
+    <Button
+      leftIcon={<IoChevronUpSharp />}
+      variant="outline"
+      isSecondary
+      onClick={scrollToTop}
+    >
+      Go to top
```

Comment:
We'll want to set this up for translation as well. I'm happy to do this before this gets merged when ready.

Reply:
oh brilliant